.github/workflows/site.yml

```diff
@@ -40,6 +40,12 @@ jobs:
       commit_sha: ${{ steps.branch.outputs.commit_sha }}
 
     steps:
+      - uses: actions/checkout@v3
+
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 16
+
       - uses: khan/pull-request-comment-trigger@master
```

Comment:
Please update this action to use This comment is for the line below: please update  to use https://github.com/marketplace/actions/pull-request-comment-trigger-fork instead instead, node version is patched there.

Reply:
We also need to update the other workflow that uses this action: https://github.com/search?q=repo%3Acheckstyle%2Fcheckstyle%20khan%2Fpull-request-comment-trigger&type=code

Reply:
@nrmancuso Updated, For the 1st warning i am creating another PR

Reply:
Leaving this unresolved for next reviewer to see.